src/scripting/game_lua_kernel.cpp

```diff
@@ -2443,15 +2595,22 @@ int game_lua_kernel::intf_play_sound(lua_State *L)
  * - Arg 1: location.
  * - Arg 2: boolean preventing scroll to fog.
  * - Arg 3: boolean specifying whether to warp instantly.
+ * - Arg 4: boolean specifying whether to skip if already onscreen
  */
 int game_lua_kernel::intf_scroll_to_tile(lua_State *L)
 {
 	map_location loc = luaW_checklocation(L, 1);
 	bool check_fogged = luaW_toboolean(L, 2);
-	bool immediate = luaW_toboolean(L, 3);
+	game_display::SCROLL_TYPE scroll = luaW_toboolean(L, 4)
+		? luaW_toboolean(L, 3)
+			? game_display::WARP
+			: game_display::SCROLL
+		: luaW_toboolean(L, 3)
```

Comment:
ok but then the releasenotes is likeley wrong, since it now accepts a fourth argument, not a third